contrib/mod_snmp/mod_snmp.c

```diff
@@ -2210,7 +2210,7 @@ MODRET snmp_pre_list(cmd_rec *cmd) {
   }
 
   proto = pr_session_get_protocol(0);
-  if (strncmp(proto, "ftp", 4) == 0) {
+  if (strcmp(proto, "ftp") == 0) {
```

Comment:
These are opportunistic cleanups; `strcmp(3)` is to be preferred when comparing whole strings.